tests/test-dirs/with-ppx.t/run.t

```diff
@@ -0,0 +1,29 @@
+  $ cat >main.ml <<EOF
+  > match Some 3 with
+  > | None -> ()
+  > | Some _ -> print_int [%get_int 42]
+  > EOF
```

Comment:
I think this test should also include an actual typed hole; perhaps `_` instead of `()`?